mcp_zammad/client.py

```
@@ -481,3 +499,332 @@ def list_tags(self) -> list[dict[str, Any]]:
+
+    # ------------------------------------------------------------------
+    # Knowledge Base methods (direct HTTP – not covered by zammad_py)
+    # Read-only operations only; writes/attachments land in follow-up PRs.
+    # ------------------------------------------------------------------
+
+    def _kb_url(self, *parts: str | int) -> str:
+        """Build a knowledge-base API URL from path components."""
+        path = "/".join(str(p) for p in parts)
+        return f"{self.api.url}knowledge_bases/{path}"
+
+    def _kb_raise_or_return(self, response: Any) -> dict[str, Any] | list[Any]:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `response`

(ANN401)